controllers/articles-controller.js

```diff
@@ -0,0 +1,18 @@
+const { getArticles } = require('../models/articles-model')
+
+const getArticlesController = (req, res, next) => {
+    const { article_id } = req.params
+    console.log(article_id)
+    if (isNaN(Number(article_id))) {
+        return res.status(400).send({msg: 'Invalid article id'})
+    }
```

Comment:
psql error will handle it for you. So leave it for psql error handler rather than creating a new customs error  (code :"22P02") Please refer to the Error handling notes.